### DISCUSSION TITLE
Customization/Branding

### DISCUSSION THREAD

**Discussion by epfaffinger** [mspnp/AzureNamingTool#127] [Q&A]
Hello, we have deployed this tool as an App Service in Azure and our company has re-branded. I was wondering if it's possible to brand/customize the look of the app, font, colours, logos, etc. Or if doing so would just be wiped out on any updates?

**Comment by epfaffinger** [comment]
Anyone have a chance to answer this question?

**Reply by BryanSoltis** [reply]
Hi @epfaffinger,

The tool does allow for Home Page Customization and Logo. You can find this in the Admin section:

![image](https://github.com/user-attachments/assets/4f156cb1-62dd-4ead-a72a-12e77175eaf3)

Note that if you are hosting in a container, the logo is not stored in the persistent storage, so you will need to add that again if the container is deleted/recreated.

-Bryan

**Comment by epfaffinger** [comment]
Thank you kindly Bryan!

**Comment by BryanSoltis** [comment]
Happy to help! For the colors, that is a bit more complex as the theme is used across the entire site. We can evaluate this feature and possibly include it in a future release.

**Comment by epfaffinger** [comment]
Thank you again Bryan, for us the color theme across the entire app is preferred so that's ok. We aren't running it in a container either so glad to hear it shouldn't be overwritten with future updates.
